frontend/src/service/Api.service.ts

```diff
@@ -0,0 +1,88 @@
+import { AuthService } from './Auth.service';
+import { Err, ErrorService } from './Error.service';
+import { EnvService } from './Env.service';
+import { GraphQLService, IGraphQLError } from './GraphQL.service';
+import { Url } from '../entity/Url';
+
+interface IGraphQLSchemaQuery {
```

Comment:
```suggestion
interface IShortGraphQLQuery {
```

Reply:
We may call GraphQL apis by other providers. So it's important to make it clear that this one is for Short.